real_preprocessing/README.md

```diff
@@ -0,0 +1,24 @@
+# real\_preprocessing
+
+### Additional packages  
+
+* [apriltag\_ros](http://wiki.ros.org/apriltag_ros)
+* [cv\_bridge](http://wiki.ros.org/cv_bridge)
+* [tf\_conversions](http://wiki.ros.org/tf_conversions)
+
+[OpenCV](https://www.learnopencv.com/install-opencv-3-4-4-on-ubuntu-16-04/) was installed on Ubuntu 16.04 and included in CMakeLists.txt as follows
+
+	find_package(find_package(OpenCV REQUIRED PATHS "YOUR-FOLDER-PATH")
+
+### Image publishing 
+**corner\_detection.cpp** simulates image capture with user input. By default, `""` /`ENTER` is the assigned trigger for image processing and YAML dumping. This can be changed at line 15, `if(capture=="")`. The YAML file for each frame detection is dumped at the working directory. To view the video stream detection, [image\_view](http://wiki.ros.org/image_view) was the library included. Rviz can be used alternatively with some configuration.
```

Comment:
Nit: If the code is meant to be user friendly then you probably should not expect them to change lines of code. It might make sense to pass in command line arguments (which is not really priority now). Either force the user to only use the ENTER key or add a TODO for the future.